### PR TITLE
Refine user interface

### PR DESCRIPTION
**Pull request by ruibaby**
Some user interface improvements

1. Use codemirror to edit text diagram code
 
    图片

2. Display mermaid error

    图片

3. Add fullscreen feature

    图片

4. Add documentation link

    图片

/kind improvements

```release-note
优化 UI 界面，提升易用性
```

### CODE REVIEW

File: console/src/editor/text-diagram/TextDiagramView.vue [hunk 1]
```diff
@@ -92,17 +115,29 @@ onMounted(() => {
...
+          v-tooltip="`查阅 ${language.label} 的文档`"
+          :href="language.document"
+          target="_blank"
+        >
+          <IcOutlineTipsAndUpdates />
+        </a>
+      </div>
+      <div class="text-diagram-nav-end">
+        <VButton size="sm" @click="fullscreen = !fullscreen">
+          {{ fullscreen ? "退出全屏" : "全屏" }}
```
Comment (JohnNiang) [inline console/src/editor/text-diagram/TextDiagramView.vue]: 这里用图标会不会更好呢？

**Comment by ruibaby** [comment]
After this, we can try adding syntax support for Mermaid and PlantUML in Codemirror. There is already a library available that supports Mermaid syntax: https://github.com/inspirnathan/codemirror-lang-mermaid

File: console/src/editor/text-diagram/TextDiagramView.vue (hunk 1 above)
Comment (ruibaby) [reply console/src/editor/text-diagram/TextDiagramView.vue]: 我试试。

**Review by JohnNiang — APPROVED** [review]
/approve

**Review by guqing — COMMENTED** [review]
/lgtm

**Pull request merged**